interfaces/Portalicious/src/app/components/form-field-wrapper/form-field-wrapper.component.html

```diff
@@ -1,6 +1,9 @@
 <!-- eslint-disable-next-line @angular-eslint/template/label-has-associated-control -->
-<label class="[&_.p-password]:block [&_input]:w-full">
-  <span class="mb-2 inline-block">{{ label() }}</span>
-  <ng-content></ng-content>
```

Comment:
The content was inside the label for a reason: this allows for the field to be connected to the appropriate label without extra hurdles (eg. making the sure the field has a `for` field that matches the `name` of the field). Connecting the field means (among other things)
- the field is selectable/focusable by clicking on the label
- screen readers correctly identify the field name

IMHO we need to revert this change.